state/api/src/main/java/com/expediagroup/streamplatform/streamregistry/state/EntityView.java

```diff
@@ -36,13 +37,6 @@
    */
   CompletableFuture<Void> load(EntityViewListener listener);
 
-  /**
-   * Commences loading the view. The returned {@link CompletableFuture} completes when the view has fully loaded.
-   *
-   * @return a future that completes when the view has fully loaded.
-   */
-  CompletableFuture<Void> load();
-
```

Comment:
Are we sure this isn't being used by any customers? This is a part of the library that is accessible to them, right?

Reply:
I noticed @nahguam's comment below so maybe he can answer this

Reply:
> The EntityView#load() can also probably be removed.

This was only used in `KafkaConfiguration`. None of our clients used it. If anyone else was using it, I think it makes sense for them to actually explicitly define which `EntityViewListener` they want to use.